debezium-core/pom.xml

```diff
@@ -30,6 +30,14 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@jpechane, so that's the one pending question: either make them optional or put the Serdes into a new module. I'd lean towards the former for the sake of simplicity. Always can extract them to a separate module later on.

Reply:
@jpechane, this still needs addressing?

Reply:
No, it is addressed at connector-distribution.xml

Reply:
Interesting, I see. Note it means though that users of the embedded engine (thus depending on _debezium-core_ as a Maven dependency, as opposed to using the connector archives in the KC deployment scenario) will have those transitive dependencies pulled in. Not a huge issue, but they won't need these in all likelyhood.

Reply:
@jpechane this one still needs convergence. I still would prefer if those dependencies were marked as optional.

Reply:
The depenedncies are optional, just `jackson-core` is not as it was not added by this PR.

Reply:
Ok, I see, by virtue of dependency management. Note I personally prefer to give the scope and optionality in actual usages for the sake of understandability (also that info may change between different usages within a multi-module project, unlike the version which always should be the same), but no need to change.